tests/test_data.py

```diff
@@ -10,18 +15,33 @@ async def test_get_tool_metadata(shared_datadir):
     os.system(f'rm -rf {tmp_dir}') # maybe not the best method?
     assert df.loc[df['name'] == 'PEPMatch', 'pmid'].values[0] == str(38110863)
 
-def test_europepmc(shared_datadir):
-    ProteoWizard_pmid = 23051804
-    citations = data.europepmc(ProteoWizard_pmid, page_size=1000)
-    assert len(citations)> 100
+
+@pytest.mark.asyncio
+async def test_europepmc_request(shared_datadir):
+    ProteinProphet_pmid = 14632076 #ProteinProphet has 2949 citations on Jul 12th 2024
```

Comment:
This variable is specified but is not being used, right? It is also good to avoid using capitalized variable names in Python. You will see often that only class names are capitalized, and all variables, module names, package names, etc. start with a small letter (exceptions are all capital variables in some cases).